### ISSUE TITLE
Configure travis to treat sphinx warnings as errors

### ISSUE THREAD

**Issue by jputrino** [F5Networks/f5-common-python#324]
Change the script for the docs build to `sphinx-build -W docs/ docs/_build/`.

This is dependent on #244; otherwise the builds will never pass.

**Comment by wojtek0806** [comment]
Issue Review: I believe we can fold this into #244

**Comment by zancas** [comment]
@jputrino  your call, close this issue and just track with #244, or leave open because it's sufficiently different.

**Comment by jputrino** [comment]
I'm fine with tracking via #244. This isn't the highest priority issue at present, since I build everything locally prior to PR; it's a nice-to-have.
